2019 카카오 개발자 겨울 인턴십/튜플/PGS_튜플_홍예진.java

```diff
@@ -0,0 +1,27 @@
+import java.util.*;
+
+class Solution {
+    public int[] solution(String s) {
+        s = s.replaceAll("\\}", "");
+        s = s.replaceAll("\\{", "");
+        StringTokenizer st = new StringTokenizer(s, ",");
+        Map<Integer, Integer> map = new HashMap<>();
+        while(st.hasMoreTokens()){
+            int key = Integer.parseInt(st.nextToken());
+            if(map.get(key) == null)
+                map.put(key, 1);
+            else 
+                map.put(key, map.get(key)+1);
```

Comment:
맵에 키가 있냐 없냐를 판단할 때는 `map.containsKey(key)`이 있어서 참고하시면 좋을 것 같아요!

```suggestion
              map.put(key, map.getOrDefault(key)+1);
```

value값을 증가시킬 때 기존에 값이 아무것도 없을 때를 고려하여 if문을 쓰신 것으로 생각하는데
`map.getOrDefault(key,defaultValue)`을 이용하여 코드를 간결화할 수 있어서 참고하시면 좋을 것 같아요!